anchor/common/qbft/src/config.rs

```diff
@@ -27,21 +25,12 @@ impl<F: Clone + LeaderFunction> Config<F> {
     pub fn operator_id(&self) -> OperatorId {
         self.operator_id
     }
-    /// The committee size
-    pub fn committee_size(&self) -> usize {
-        self.committee_size
-    }
-
-    pub fn commmittee_members(&self) -> HashSet<OperatorId> {
-        self.committee_members.clone()
-    }
 
-    /// The quorum size required for the committee to reach consensus
-    pub fn quorum_size(&self) -> usize {
-        self.quorum_size
+    pub fn commmittee_members(&self) -> &IndexSet<OperatorId> {
```

Comment:
We could add this to the beginning of this `impl`

```
    /// Private constructor so it can only be built by our `ConfigBuilder`.
    fn from_builder(builder: &ConfigBuilder<F>) -> Self {
        Self {
            operator_id: builder.operator_id,
            instance_height: builder.instance_height,
            committee_members: builder.committee_members.clone(),
            round: builder.round,
            round_time: builder.round_time,
            max_rounds: builder.max_rounds,
            quorum_size: builder.quorum_size,
            leader_fn: builder.leader_fn.clone(),
        }
    }
```